src/module/dy/validate-kwargs.js

```diff
@@ -59,5 +57,5 @@ module.exports = (kwargs) => {
   if (result.error) {
     throw new Error(result.error);
   }
-  return result;
+  return result.value;
```

Comment:
There was an issue in create-model with this. Made some improvements to the tests below.